### PR TITLE
Added Event in_span scope

### PR DESCRIPTION
**Pull request by mhib**
@quirk0o @adamzima @s1mplex

### CODE REVIEW

File: spec/models/event_spec.rb [hunk 1]
```diff
@@ -36,8 +36,17 @@
...
+      expect(described_class.in_span(start_time.beginning_of_week, start_time.end_of_week)).to match_array expected_events
     end
+  end
+
+  describe '.in_week' do
 
+    let(:start_time) { Time.now.beginning_of_week }
+
+    it 'calls. in_span with correct arguemnts' do
+      expect(described_class).to receive(:in_span).with(start_time.beginning_of_week, start_time.end_of_week)
```
Comment (mhib) [inline spec/models/event_spec.rb]: @adamzima

File: app/models/event.rb [hunk 2]
```diff
@@ -9,11 +9,14 @@ def start_time_must_be_lower_than_end_time
     errors.add(:start_time, "Can't be higher than end_time") if start_time.try(:>, end_time)
   end
 
+  scope :in_span, -> (starting, ending) {
+    where('start_time >= ? AND start_time <= ?', starting, ending)
```
Comment (s1mplex) [inline app/models/event.rb]: `where('? <= end_time AND ? >= start_time', starting, ending)` should be equivalent to your more complex condition

Comment (mhib) [inline app/models/event.rb]: @s1mplex Done

File: spec/models/event_spec.rb [hunk 3]
```diff
@@ -36,8 +36,17 @@
...
-      expect(described_class.in_week(start_time)).to match_array expected_events
+      expect(described_class.in_span(start_time.beginning_of_week, start_time.end_of_week)).to match_array expected_events
     end
+  end
+
+  describe '.in_week' do
 
+    let(:start_time) { Time.now.beginning_of_week }
+
+    it 'calls. in_span with correct arguemnts' do
```
Comment (adamzima) [inline spec/models/event_spec.rb]: `calls .in_span`

**Comment by adamzima** [comment]
:+1:

**Comment by pbzymek** [comment]
looks good to merge

**Comment by lkudyba** [comment]
👍

**Pull request merged**